WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dashboard/todaysstats/TodaysStatsCardViewHolder.kt

```diff
@@ -99,23 +95,6 @@ class TodaysStatsCardViewHolder(
         )
     }
 
-    private fun SpannableString.withExternalLinkImageSpan(endIndex: Int) {
-        val drawable = ContextCompat.getDrawable(itemView.context, drawable.ic_external_white_24dp) ?: return
-        drawable.setTint(linkColor)
-        drawable.setBounds(
-                EXTERNAL_LINK_DRAWABLE_BOUND_LEFT,
-                0,
-                drawable.intrinsicWidth / 2,
-                drawable.intrinsicHeight / 2
-        )
-        setSpan(
-                ImageSpan(drawable, ImageSpan.ALIGN_BASELINE),
-                endIndex,
-                endIndex + 1,
```

Comment:
Some notes for anyone who lands on this in the future, especially when we might want to properly address/reimplement that (for the next release next sprint?):
 - That code made the app crash in locales where there were no character after the link in the translation, for example in French.
 - Slack thread with details and investigation discussion: p1651252336966119/1651250182.578219-slack-C011BKNU1V5
 - So if we want to re-introduce that drawable in a future release, we'll need to be sure:
    - To not fall into the same assumption trap that there would always be a character after the link (that the drawable would eat)
    - To test how the addition of the drawable after the link renders in other locales, especially in RTL locales like Korean, Japanese or Chinese, and test that the added drawable in the link span does not eat any character from the translated text.

Reply:
@AliSoftware Thanks for this note. 🙌🏼